src/utils/paymentQRHelper.ts

```diff
@@ -0,0 +1,72 @@
+import { parseAndValidateSGQR } from "@rationally-app/payment-qr-parser";
+import { Transaction } from "../types";
+
+/**
+ * Returns the value of the first element in the provided record key.
+ * If the provided record key does not exist, `undefined` is returned.
```

Comment:
small nit, should we say that this function supports maximum 2 layer of nesting in the docstring haha

Reply:
Good point, I should write tests for more than 2 layers of nesting as well, that would convey this as well.

Reply:
OH LOL, wait, you wrote a recursive function :x HAHAHa thats why it can support 3 layers, thanks for adding the tests xD